index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>My resume</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <header>
+        <img src="img/profile_photo.jpg" height="69" width="70" alt="Profile Photo"/>
+        <p>Dzmitry Karachun<br><span>My profile</span></p>
+    </header>
+    <div class="container">
+        <div class="line_1">
+            <ul class="line_2">
```

Comment:
more meaningful class name should be used instead of "line_2" class (for ex. list__item-title, itemTitle),
it's not the case for using ul tag here (list that has only 1 item is probably not a list)